File: app/src/main/java/sticky/header/tableview/stickyheadertableview/DisplayMatrixHelper.java
package sticky.header.tableview.stickyheadertableview;

import android.content.Context;
import android.util.DisplayMetrics;
import android.util.TypedValue;

public class DisplayMatrixHelper {

    public float dpToPixels(Context context, float dpValue) {
        DisplayMetrics metrics = context.getResources().getDisplayMetrics();
        return TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, dpValue, metrics);
    }
}

File: app/src/main/java/sticky/header/tableview/stickyheadertableview/DisplayMatrixHelper.kt
package sticky.header.tableview.stickyheadertableview

import android.content.Context
import android.util.TypedValue

class DisplayMatrixHelper {
    fun dpToPixels(context: Context, dpValue: Float): Float {
        val metrics = context.resources.displayMetrics
        return TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, dpValue, metrics)
    }
}